nextflow.config

```diff
@@ -156,7 +154,7 @@ params {
 
 env { 
     CONTROL_WORKFLOW_ROOT = "${baseDir}"
-    CONTROL_WORKFLOW_BRANCH = 'origin/develop'
+    CONTROL_WORKFLOW_BRANCH = 'origin/feature/cross_validation'
```

Comment:
If this is production(ish) code you shouldn't be using a feature branch of the child workflow (it should be parameterised to deal with your use case). Remember that feature branches are ephemeral- we shouldn't expect them to be around for long.

Reply:
Forgot to change this after merging the feature branch, thanks for pointing out.